src/test/java/net/minecraftforge/debug/world/ServerWorldCreationTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.world;
+
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.world.BiomeLoadingEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@Mod(ServerWorldCreationTest.MODID)
+public class ServerWorldCreationTest {
```

Comment:
Please provide a javadoc for this class which explains what feature this tests, how to test it, and what the expected results are from said test.

Also, formatting: braces on newlines.
```suggestion
public class ServerWorldCreationTest
{
```